go.mod

```diff
@@ -301,3 +302,7 @@ replace (
 	github.com/pingcap/tidb/parser => ./parser
 	go.opencensus.io => go.opencensus.io v0.23.1-0.20220331163232-052120675fac
 )
+
+replace github.com/pingcap/kvproto => github.com/pingcap/kvproto v0.0.0-20230614050532-13569312829f
+
+replace github.com/pingcap/tipb => github.com/pingcap/tipb v0.0.0-20230607071926-bda24015c2d6
```

Comment:
This looks a bit strange?

Reply:
Yeah, will update the final go mod info when ready to merge